src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -858,6 +860,32 @@ public function testInstanceof()
         $this->assertSame(['foo' => [[]], 'bar' => [[]]], $definition->getTags());
     }
 
+    /**
+     * @requires PHP >= 8.1
```

Comment:
```suggestion
     * @requires PHP 8.1
```

Let's keep this annotation consistent with the rest of the codebase. Please also change the other occurrences.

Reply:
Alright, doing it in #40830 as well